src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTypeCheckTest.java

```diff
@@ -309,7 +309,7 @@ public void testAllowMissingTypeParameters() throws Exception {
         final String[] expected = {
             "7:4: " + getCheckMessage(MSG_UNUSED_TAG, "@param", "<D123>"),
             "44:8: " + getCheckMessage(MSG_UNUSED_TAG, "@param", "<C>"),
-            "60:5: " + getCheckMessage(MSG_UNUSED_TAG, "@param", "<x>"),
+            "60:5: " + getCheckMessage(MSG_UNUSED_TAG, "@param", "x"),
```

Comment:
please move this commit to functional 
at build on it result in 
`JavadocTypeCheckTest.testAllowMissingTypeParameters:314->AbstractModuleTestSupport.verify:229->AbstractModuleTestSupport.verify:267->AbstractModuleTestSupport.verify:304 error message 1 expected:<...sed @param tag for '[x]'.> but was:<...sed @param tag for '[<x>]'.>`

All commits are atomic change that are build-able and even can be released. code should be build and function properly on each commit.

Reply:
Done.  I had forgotten about that, apologies.